src/components/ListView.js

```diff
@@ -23,12 +42,21 @@ const ListView = () => {
         {value && (
           <ul>
             {value.docs.map((groceryItem) => (
-              <li
-                key={groceryItem.id}
-                onClick={() => deleteItemHandler(groceryItem.id)}
-              >
-                {groceryItem.data().itemName}
-              </li>
+              <Fragment key={groceryItem.id}>
+                <input
+                  type="checkbox"
+                  id={groceryItem.data().itemName}
+                  name={groceryItem.data().itemName}
+                  value={groceryItem.id}
+                  onChange={onPurchaseChange}
+                  checked={checkIfItemHasBeenPurchased(
```

Comment:
This seems to be working great. I'm just curious how the checked... check works? I see how you are creating the 24 hour check, but what freezes it here? Again, just curious. It satisfies the AC super well. :)